siliconcompiler/core.py

```diff
@@ -426,7 +426,11 @@ def create_cmdline(self, progname, description=None, switchlist=None, input_map=
             if 'fpga_partname' in cmdargs.keys():
                 self.set('fpga', 'partname', cmdargs['fpga_partname'], clobber=True)
             # running target command
-            self.load_target(cmdargs['option_target'])
+            try:
+                target_module = importlib.import_module(f'targets.{cmdargs["option_target"]}')
```

Comment:
Add a TODO around this, I think we need to "find" this module in PYTHON_PATH or in targets.{}

Reply:
Good point - users could have their own targets in different locations, so we might also want to import the command-line target string literally (e.g. `sc -target targets.freepdk45_demo` instead of `sc -target freepdk45_demo`)